packages/collapsible/src/LionCollapsible.js

```diff
@@ -76,6 +72,26 @@ export class LionCollapsible extends LitElement {
     }
   }
 
+  toggle() {
+    this.__toggle();
+  }
+
+  expand() {
```

Comment:
Could we rename this to `show`, to stay inline with our other components? (e.g. overlays)

Reply:
Sure 👍